src/less/components/button.less

```diff
@@ -99,7 +99,6 @@
         font-weight: normal;
         background: transparent;
         border-radius: 0;
-        border-bottom: 2px solid @blue;
 
         // Do not change these, or the border will look weird [EH]
         padding: @padding-base-vertical 0 0;
```

Comment:
Since you are removing the border bottom for link buttons, you can probably remove this line and give `btn-link` the same padding as the rest of the buttons